src/main/kotlin/net/leanix/githubagent/dto/ManifestFilesDTO.kt

```diff
@@ -0,0 +1,12 @@
+package net.leanix.githubagent.dto
+
+data class ManifestFilesDTO(
+    val repositoryId: String,
+    val repositoryFullName: String,
+    val manifestFiles: List<ManifestFileDTO>
+)
+
+data class ManifestFileDTO(
+    val path: String,
```

Comment:
is it the para or the complete url?

Reply:
Today, the path is enough to mount the path to the file on the backend.
Repository url(already persisted on the backend) + path